homeassistant/components/ecobee.py

```diff
@@ -67,7 +75,7 @@ def setup_ecobee(hass, network, config):
         configurator = get_component('configurator')
         configurator.request_done(_CONFIGURING.pop('ecobee'))
 
-    hold_temp = config[DOMAIN].get(HOLD_TEMP, False)
+    hold_temp = config[DOMAIN].get(CONF_HOLD_TEMP, False)
```

Comment:
Please specify defaults inside the schema: `vol.Optional(CONF_HOLD_TEMP, default=False): bool`